apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.cs

```diff
@@ -37,6 +37,9 @@ namespace Google.Cloud.Storage.V1
     /// </remarks>
     public sealed partial class UrlSigner
     {
+        private const string InfiniteExpiryObsoleteMessage =
+            "This overload is obsolete, as non-expiring signed URLs are no longer supported. Please use the overload accepting a DateTimeOffset instead.";
+
```

Comment:
Technically they will still be supported until the bug is fixed. Is it more that the C# client will no longer support it until it is fixed. Or is this considered too verbose?

Suggestion:
> This overload is obsolete, as non-expiring signed URLs are a bug in the service that can cause security issues and should no longer be used. Please use the overload accepting a DateTimeOffset instead.

Reply:
Fine by me. I wasn't sure how we wanted to message the service change. Done.

(We'll want to change the documentation again a bit in the second phase where we move to making "non-expiring" just expire in 2038, but that can be on the parameter.)

Reply:
Sg, thank you!